src/components/panel/panel.js

```diff
@@ -1950,25 +1954,36 @@ MdPanelPosition.prototype._calculatePanelPosition = function(panelEl, position)
   var targetRight = targetBounds.right;
   var targetWidth = targetBounds.width;
 
+  function reverseXPosition (position) {
+    var start = 'start';
```

Comment:
Perhaps bail early if position === xPosition.CENTER, since that shouldn't matter with bidi.